user/deployment/elasticbeanstalk.md

```diff
@@ -40,9 +40,17 @@ To create an application without deploying it, add `only_create_app_version: "tr
 
 ## Optional settings
 
-* `zip_file`: The zip file to deploy. You also need to set `skip_cleanup` to prevent Travis CI deleting the zip file at the end of the build. If this is left unspecified, a zip file will be created from all the files that are part of the repository under test (determined with `git ls-files`).
+* `zip_file`: The zip file to deploy. You also need to set `skip_cleanup` to prevent Travis CI deleting the zip file at the end of the build. If this is left unspecified, a zip file will be created.
```

Comment:
> If this is left unspecified, a zip file will be created. 

sounds pretty vague. How about

If you don't specify a zip file, the entire repository will be zipped and uploaded, according to [.ebignore` or `.gitignore` rules](...)
?